src/device.coffee

```diff
@@ -186,7 +186,8 @@ exports.getOSVersion = ->
 		for line in lines
 			[ key, val ] = line.split('=')
 			releaseItems[_.trim(key)] = _.trim(val)
-		return releaseItems['PRETTY_NAME']
+		# Remove enclosing quotes: http://stackoverflow.com/a/19156197/2549019
+		return releaseItems['PRETTY_NAME'].replace(/^"(.+(?="$))"$/, '$1')
```

Comment:
Your examples made me think I'm not exactly sure why that file has quotes and what the supposed syntax is, are there escaped quotes inside the quotes for example?

In any case, LGTM if you know that the syntax either has quotes at start and end or you don't care.